src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/EmbeddedMongo.scala

```diff
@@ -11,7 +11,7 @@ object EmbeddedMongo extends MongoEmbedDatabase {
   def startMongo() = {
     println(s"Starting embedded mongo db instance.")
     if (mongodProps == null || !mongodProps.mongodProcess.isProcessRunning) {
-      mongodProps = mongoStart(port = 27017, version = Version.V3_4_1)
+      mongodProps = mongoStart(port = 27017, version = Version.V3_0_6)
```

Comment:
in prod, our mongo version is 3.0.6, so let's match that here. I realize this gives a deprecation warning, but in this particular case I think that's good ... it'll prod us to upgrade (GAWB-3037)